_saas-integrations/recharge/v1/recharge-v1.md

```diff
@@ -0,0 +1,133 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/integration-templates/saas/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#      Page & Formatting     #
+# -------------------------- #
+
+title: ReCharge
+permalink: /integrations/saas/recharge
+keywords: recharge, integration, schema, etl recharge, recharge etl, recharge schema
+layout: singer
+# input: false
+
+key: "recharge-setup"
+
+
+# -------------------------- #
+#         Tap Details        #
+# -------------------------- #
+
+name: "recharge"
+display_name: "ReCharge"
+
+singer: true
+status-url: ""
+
+tap-name: "recharge"
+repo-url: https://github.com/singer-io/tap-recharge
+
+this-version: "1.0"
+
+api: |
+  [ReCharge Payments API](https://developer.rechargepayments.com/){:target="new"}
+
+
+# -------------------------- #
+#       Stitch Details       #
+# -------------------------- #
+
+certified: true 
+
```

Comment:
Another one I haven't added to the metadocs yet - a property named `api-type`.

The value should match whatever is in the Connect source form property for the integration. So, in this case:

```
api-type: "platform.recharge"
```

This should be present for any integration that's supported in the Connect API. I'll add it to the metadocs as well.